### PR TITLE
task 8 is ready for review

### PR DESCRIPTION
**Pull request by Hanieh73**
task8 is done for PATCH /api/articles/:article_id and ready for review

### CODE REVIEW

File: controller/articles.controller.js [hunk 1]
```diff
@@ -60,3 +60,20 @@ exports.postComment = (req, res, next) => {
...
 }
+
+exports.patchArticle = (req, res, next) => {
+    const { article_id } = req.params
+    const { inc_votes } = req.body
+  
+    patchArticleVoteById(article_id, inc_votes)
+        .then((article) => {
+            res.status(200)
+            .send(article)
```
Comment (Jodiech) [inline controller/articles.controller.js]: You should return the data on an appropriate key eg {article} 👍

File: models/articles.model.js [hunk 2]
```diff
@@ -62,6 +62,29 @@ exports.postCommentByArticleId = (article_id, commentData) => {
...
+    SET
+    votes = votes + $1
+    WHERE article_id = $2
+    RETURNING *;
+    
+    `, [inc_votes, article_id]).then((response) => {
+        
+ 
+        
+        if (response.rows.length === 0 || !article_id) {
```
Comment (Jodiech) [inline models/articles.model.js:78]: Nice work here 👍

**Review by Jodiech — APPROVED** [review]
Great job on this ticket Hanieh. Please make sure you are returning the data back with the appropriate key (please see comment below). Please merge once you have changed this 😄

File: controller/articles.controller.js (hunk 1 above)
Comment (Hanieh73) [reply controller/articles.controller.js]: done :)

**Pull request merged**